File: DotNet/CesiumLanguageWriter/Advanced/ICesiumBooleanValuePropertyWriter.cs
namespace CesiumLanguageWriter.Advanced
{
    /// <summary>
    /// A writer that can write a value as a boolean.
    /// </summary>
    public interface ICesiumBooleanValuePropertyWriter : ICesiumPropertyWriter
    {
        /// <summary>
        /// Writes the value expressed as a boolean value.
        /// </summary>
        /// <param name="value">The value.</param>
        void WriteBoolean(bool value);
    }
}

File: Java/CesiumLanguageWriter/translatedSrc/cesiumlanguagewriter/advanced/ICesiumBooleanValuePropertyWriter.java
package cesiumlanguagewriter.advanced;


import agi.foundation.compatibility.*;
import cesiumlanguagewriter.*;

/**
 * A writer that can write a value as a boolean.
 */
@SuppressWarnings({
    "unused",
    "deprecation",
    "serial"
})
public interface ICesiumBooleanValuePropertyWriter extends ICesiumPropertyWriter {
    /**
    * Writes the value expressed as a boolean value.
    * @param value The value.
    */
    void writeBoolean(boolean value);
}